### ISSUE TITLE
How long does a query last?

### ISSUE THREAD

**Issue by dayer3** [ddavison/rest-client#73]
Hi. I think it could be interesting know how long a query last.
Is this implemented? Do you agree?

**Comment by ddavison** [comment]
do you mean like a progress bar, for the request?

**Comment by dayer3** [comment]
Not necessarily. Show the total time of the request is sufficient, imho.

**Comment by ddavison** [comment]
I agree this would be nice.  if you wanted to see if you could tackle it, go ahead and submit a pull request! otherwise - you'll have to wait until one of us gets to it.

👍
